docs/images.md

```diff
@@ -99,7 +99,7 @@ These approaches provide no safety checks. It's up to you to guarantee that thos
 
 ## Network Images
 
-Many of the images you will display in your app will not be available at compile time, or you will want to load some dynamically to keep the binary size down. Unlike with static resources, _you will need to manually specify the dimensions of your image_. It's highly recommended that you use https as well in order to satisfy [App Transport Security](running-on-device.md#app-transport-security) requirements on iOS.
+Many of the images you will display in your app will not be available at compile time, or you will want to load some dynamically to keep the binary size down. Unlike with static resources, _you will need to manually specify the dimensions of your image_. It's highly recommended that you use https as well in order to satisfy [App Transport Security](integration-with-existing-apps.md#app-transport-security) requirements on iOS.
```

Comment:
The link https://reactnative.dev/docs/integration-with-existing-apps#app-transport-security

Seems broken to me (the anchor is not working).

Perhaps it should be:
https://reactnative.dev/docs/publishing-to-app-store#1-enable-app-transport-security

Reply:
Yeah i just noticed that also, i think that is the proper link [https://reactnative.dev/docs/publishing-to-app-store#1-enable-app-transport-security](https://reactnative.dev/docs/publishing-to-app-store#1-enable-app-transport-security). 

Interesting notes because of this `app-transport-security` reference on `network.md` is also broken. Should i also update the reference there?

Reply:
Yup let's update them all 👍

Reply:
Will do @cortinico. Should we update the versioned docs too?

Reply:
Ideally yes, till the versions where this change got introduced

Reply:
okay go it, let me check at what version of the docs does this change got introduced and i will update the docs again. Thanks @cortinico

Reply:
@cortinico got the pr reference for the change.

Reference PR 
https://github.com/facebook/react-native-website/pull/2162

The section `### 1. Enable App Transport Security` is available on `publishing-to-app-store` all the way to version `0.60`.

Reply:
Great, then we can replicate the changes to all the versions we have (which is till 0.60)